collectors/gsuite/Makefile

```diff
@@ -0,0 +1,40 @@
+AWS_LAMBDA_FUNCTION_NAME ?= alertlogic-gsuite-collector
+AWS_LAMBDA_PACKAGE_NAME ?= al-gsuite-collector.zip
+AWS_LAMBDA_S3_BUCKET ?= alertlogic-collectors
+
+.PHONY: test
```

Comment:
could you change this makefile to use the common template like in o365?

https://github.com/alertlogic/paws-collector/blob/master/collectors/o365/Makefile